src/main/java/org/galatea/starter/service/IexService.java

```diff
@@ -19,9 +23,8 @@
 public class IexService {
 
   @NonNull
+  private IexCloudClient iexCloudClient;
   private IexClient iexClient;
```

Comment:
add another @NonNull here. it only applies to the field it is directly on top of